### DISCUSSION TITLE
JSON Schema Viewer in Documentation ?

### DISCUSSION THREAD

**Discussion by jy95** [wailsapp/wails#2728] [Ideas]
Hello,

I noticed on https://wails.io/docs/reference/project-config/ that you have a JSON Schema .
I think that [docusaurus-json-schema-plugin](https://github.com/jy95/docusaurus-json-schema-plugin) might be a interesting way to explore config in a dynamic way.

Documentation & examples on https://jy95.github.io/docusaurus-json-schema-plugin/

Kind regards,

**Comment by leaanthony** [comment]
Thanks for the tip!
